src/lib/backend.ts

```diff
@@ -339,14 +339,23 @@ export class ElectronBackend implements Backend {
 
   /** Activates the Node SDK for the main process. */
   private async installMainHandler(): Promise<boolean> {
+    const options = this.frontend.getOptions();
+    // tslint:disable-next-line:no-unbound-method
+    const customHandler = options.onFatalError;
+
     await this.frontend.setOptions({
       onFatalError: async (error: Error) => {
-        console.error('*********************************');
-        console.error('* SentryElectron unhandledError *');
-        console.error('*********************************');
-        console.error(error);
-        console.error('---------------------------------');
         await this.frontend.captureException(error);
```

Comment:
It is a good idea to always log the error. However, this implementation makes it hard to understand what is happening due to its side effects. I'd rather prefer a simpler (though not equal) fix for now, as it was originally intended:

```ts
    if (!this.frontend.getOptions().onFatalError) {
      await this.frontend.setOptions({
        onFatalError: async (error: Error) => {
          console.error('*********************************');
          console.error('* SentryElectron unhandledError *');
          console.error('*********************************');
          console.error(error);
          console.error('---------------------------------');
          await this.frontend.captureException(error);
        },
      });
    }
```